Content.Client/Chemistry/UI/ChemMasterBoundUserInterface.cs

```diff
@@ -36,20 +36,20 @@ protected override void Open()
             _window.OpenCentered();
             _window.OnClose += Close;
 
-            //Setup static button actions.
-            _window.EjectButton.OnPressed += _ => SendMessage(new ItemSlotButtonPressedEvent(BeakerSlotId));
-            _window.BufferTransferButton.OnPressed += _ => PrepareData(UiAction.Transfer, null, null, null, null, null);
-            _window.BufferDiscardButton.OnPressed += _ => PrepareData(UiAction.Discard, null, null, null, null, null);
-            _window.CreatePillButton.OnPressed += _ => PrepareData(UiAction.CreatePills, null, _window.LabelLine, null, _window.PillAmount.Value, null);
-            _window.CreateBottleButton.OnPressed += _ => PrepareData(UiAction.CreateBottles, null, _window.LabelLine, null, null, _window.BottleAmount.Value);
+            // Setup static button actions.
+            _window.EjectButton.OnPressed += _ => SendMessage(new ItemSlotButtonPressedEvent("container_slot"));
```

Comment:
this needs to use a static string like it did before or someone is going to forget to change this